resources/external_control.urscript

```diff
@@ -298,6 +305,31 @@ def checkSlowDownRequired(x, qd, max_speed, max_deceleration):
   return False
 end
 
+###
+# @brief Find the maximum value in a list the list must be of non-zero length and contain numbers
+# @param list array list
+###
+def list_max_norm(list):
+  # ensure we have something to iterate over
+  local length = get_list_length(list)
+  if  length == 0:
+      popup("Getting the maximum of an empty list is impossible in list_max().", error = True, blocking = True)
```

Comment:
Not sure that this is the best way to handle the case of receiving an empty list.
Maybe ignore the trajectory instead (with an error message) and let the driver continue to be ready to receive the next command.